### DISCUSSION TITLE
"We weren't able to create this release for you. Make sure you have a valid tag."

### DISCUSSION THREAD

**Discussion by warrenlr** [community/community#29170] [Other Feature Feedback, Questions, & Ideas] [answered]
Anyone else getting this message while attempting to publish a release?

`We weren’t able to create the release for you. Make sure you have a valid tag.`

I tried to publish a release half a dozen times this morning.

I looked at my tag rules and they are v*, so it should work.  Never before encountered this, and it's a blocker.
![Screen Shot 2022-08-11 at 8 03 38 AM](https://user-images.githubusercontent.com/5719157/184166508-013bd322-7d83-4cda-ab85-c3fa47686dce.png)

**Comment by radar-blip** [comment, marked as answer]
The new tag name should be suggested for you under the 'Choose a tag' tab. Try clicking on that tab after you create the new tag name.

**Reply by stevebroshar** [reply]
hmm. I don't get it. "Choose a tag" is not a tab. It's dropdown. And what drops down doesn't seem useful to me.

![image](https://user-images.githubusercontent.com/4449426/208166950-ffca6ecd-5c7c-4a3f-aff6-65f3b39f98e4.png)

**Reply by neerajkumar161** [reply]
It worked @linuxlawson. Thank you :rocket: :smiley:

**Reply by radar-blip** [reply]
You're welcome! 👍🏻

**Reply by danielPaulinoMesquita** [reply]
Thank you for your support. I had the same problem, but it has been resolved now.

**Reply by lakesare** [reply]
<img width="360" alt="image" src="https://github.com/community/community/assets/7578559/9cb26b86-a82a-4388-9248-3baf1aa7e2d4">

You have to actually *type* your new tag name in that dropdown.

**Reply by winrid** [reply]
this is absolutely terrible UX! they should fix this.

**Reply by sankxSK** [reply]
i have problem that there circle loading to infinitely
![image](https://github.com/community/community/assets/95546134/4022afc6-d12f-4f3e-9319-8215e7cf5338)
here you can see screenshot, i left pc for 5 hours and it was still there.
can someone tell me how to fix it or anything?

**Reply by zeeshandev15** [reply]
thanks dear

**Reply by michaelschuett-tomtom** [reply]
wow. This is absolutely terrible UI/UX.

**Reply by Imzizu** [reply]
Follow these steps if you've already created a repository on the GitHub website and now want to connect your local code to it.

1. **Initialize Git in your folder.** This command creates a hidden `.git` subfolder that lets Git start tracking your project files. In your command line, make sure you are still inside your project folder (`C:\Users\zidaa\AndroidStudioProjects\chooseicon`) and run:
    
    Bash
    
    ```
    git init
    ```
    
2. **Connect your folder to the GitHub repository.** You need to get the URL from your GitHub repository page. It's the one you use to clone. Click the green **< > Code** button and copy the HTTPS URL. It will look something like `https://github.com/your-username/chooseicon.git`.
    
    Now, run this command, pasting your URL at the end:
    
    Bash
    
    ```
    git remote add origin https://github.com/your-username/chooseicon.git
    ```
    
3. **Add and save your files.** These two commands prepare and save all your project files into Git's history.
    
    Bash
    
    ```
    git add .
    git commit -m "First commit"
    ```
    
4. **Push your code to GitHub.** This uploads your saved files to the GitHub website.
    
    Bash
    
    ```
    git push -u origin main
    ```
    
    _(Note: Your main branch might be called `master` instead of `main` if it's an older repository. If `main` doesn't work, try `master`.)_
    
5. **Now, try tagging again!** ✅ You're all set up. Now you can run the original commands to create and upload the tag.
    
    Bash
    
    ```
    git tag v1.0
    git push --tags
    ```

**Comment by warrenlr** [comment]
Interesting, never had to do this before (I recall the version name would come from the text box "Release title".  
Thank you @linuxlawson, that did it (i.e. by first creating the tag by clicking on the drop down "Choose a tag", typing the new version and selecting it.

**Reply by radar-blip** [reply]
Right. I think it might be something new, same thing happened to me just recently. 
Anyway, glad it worked!

**Reply by Shmilysw** [reply]
I also encountered the same problem, thank you for your answer.

**Comment by edwardCao** [comment]
This is a defect. As the release editing page says under the "Choose a Tag" drop-down that "Choose an existing tag, or create a new tag on publish". This wording means that you don't have to manually create a new tag from the drop-down. What you need is just to put in the new tag name in the text box and upon clicking Publish the new tag is automatically created for you. That was the behavior of the previous versions of github.

**Reply by JohnTasler** [reply]
I'm having the same problem today. I didn't have this problem 4 days ago.

**Comment by kuldeep3** [comment]
When I click on create discussion for this release while creating release, it everytime throws this error
`We weren't able to create a discussion for this release. Make sure the release has a title and description`

Any thing that I may be missing?

<img width="958" alt="Screenshot 2022-09-28 at 7 53 03 PM" src="https://user-images.githubusercontent.com/37093310/192804670-2e896f9f-e237-4b15-a449-057f18310873.png">

**Comment by lharland** [comment]
It confused me when the error occurred. Create a new tag only occurs if something was typed in it.

**Reply by coindegen** [reply]
agreed, this is horribly confusing UX. 

To be clear, you can't just click the "Create new tag" button, you have to fill out the input then click it.

<img width="322" alt="image" src="https://user-images.githubusercontent.com/87461192/198423631-b5016a84-2d0c-40a6-9d11-e94cde31ab86.png">

**Reply by wwc7654321** [reply]
thanks, it's a confusing design

**Reply by musta20** [reply]
Can you imagine it's 2024 and still no change to the UX

**Reply by stevebroshar** [reply]
> Can you imagine it's 2024 and still no change to the UX

Well, it's not uncommon. There are many bad ux issues in Microsoft Word and Outlook that have been there for decades.

**Comment by neerajkumar161** [comment]
Same thing with me, when I tried to create a tag its redirect me to the previous page i.e releases page

**Comment by kulasekara02** [comment]
> If it does not exist, you need to create a new tag.
> If the tag exists, make sure you are using the correct tag name and spelling when creating the release.


Once you have verified that you have a valid tag, try creating the release again.  If you are still experiencing issues, you can try creating the release from the command line using the Git command `"git tag -a <tagname> -m 'message'"` followed by `"git push --tags`". This will create a new tag and push it to the remote repository.

![image](https://user-images.githubusercontent.com/57080664/230339007-cf5b640d-b9cb-44ba-ac2d-6ace3f943138.png)

**Reply by ermjs** [reply]
Writing `v1.0.0 ` for `Coose a tag` drop down menu worked for me. Thanks.

**Reply by mpaulasg** [reply]
Thanks! This worked for me too

**Reply by kulasekara02** [reply]
> Writing `v1.0.0 ` for `Coose a tag` drop down menu worked for me. Thanks.

(=

**Comment by bsbodden** [comment]
Github UX is typically pretty intuitive, this particular item is not.

**Comment by JorensM** [comment]
You have to specify a tag name as shown in [this reply](https://github.com/orgs/community/discussions/29170#discussioncomment-5541875) 

1. Open the **choose tag** dropdown
2. Enter a name for your tag
3. Click on **create new tag**

**Comment by gameix** [comment]
i created a tag via my IDE (Intellij) and push the tag, afterward i could select the TAG and crate the release...

**Comment by drewclifton** [comment]
GH newbie.... Wait I don't understand - we need to create a version title and a version tag? If the version tag is supposed to be the version number, what am I supposed to put for the title?

I see I don't actually need a title, hehe. What "should" go there, like a short description of what was done in this release? Or just some generic text like "Updates" or something? Thanks!
